UE/Application/Ports/UserPort.cpp

```diff
@@ -140,9 +145,44 @@ common::PhoneNumber UserPort::getInputPhoneNumber(IUeGui::ISmsComposeMode& compo
     return composer.getPhoneNumber();
 }
 
+common::PhoneNumber UserPort::getInputPhoneNumber(IUeGui::IDialMode &dial)
+{
+    return  dial.getPhoneNumber();
+}
+
 std::string UserPort::getInputString(IUeGui::ISmsComposeMode& composer)
 {
     return composer.getSmsText();
 }
 
+
+void UserPort::showNewCallRequest(common::PhoneNumber from) {
+    IUeGui::ITextMode& incomingCall = gui.setAlertMode();
+    incomingCall.setText("Incoming call from " + to_string(from));
+}
+
+void UserPort::showTalking()
+{
+    IUeGui::ICallMode& call = gui.setCallMode();
+
+}
+
+void UserPort::showDialing(common::PhoneNumber to)
+{
+    IUeGui::ITextMode& dialing = gui.setAlertMode();
+    dialing.setText("Dialling to " + to_string(to) + "...");
```

Comment:
```suggestion
    dialing.setText("Dialing to " + to_string(to) + "...");
```
dialing is US English, dialling UK English. let's use only US version for consistency